packages/wdio-browser-runner/a.js

```diff
@@ -0,0 +1,3 @@
+import { resolve } from 'import-meta-resolve'
```

Comment:
where is this file used?

Reply:
it was a test file of mine, removed in https://github.com/webdriverio/webdriverio/commit/b8675e9317dbdb42ab52f9965dfdf4cb30ed8eaf